public/js/components/App.js

```diff
@@ -35,17 +35,26 @@ const App = React.createClass({
   componentDidMount() {
     this.shortcuts = new KeyShortcuts({ window });
     this.shortcuts.on("Cmd+P", this.toggleSourcesSearch);
+    window.addEventListener("keydown", this.onKeyDown);
```

Comment:
I tried `this.shortcuts.on("Escape")` but it threw an exception. I then figured maybe it's only for keys with modifiers. Wasn't sure why the exception